.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: node_js
 node_js:
-  - "10"
+  - "12"
```

Comment:
12 starts active LTS on 2019-10-22, https://github.com/nodejs/Release

there is nothing forcing us to update, would like to wait until at least vscode ships with node 12

Reply:
ok, will fix

Reply:
fixed